.github/actions/deploy-ubuntu/action.yml

```diff
@@ -78,7 +80,9 @@ runs:
         source /etc/os-release
         export CODENAME=$UBUNTU_CODENAME
 
-        if [[ ! "$ARCH" == "amd64" ]]; then
+        # arm64 runners don't work with apt-spy2,
+        # the selected mirrors don't have arm64 binaries
+        if [[ ! "$ARCH" == "amd64" && ! "$ARCH" == "arm64" ]]; then
```

Comment:
We need this when cross compiling though. Please disable it only when we're running on ARM

Reply:
Will do